website/examples/default-theme/README.md

```diff
@@ -1,21 +0,0 @@
-# Algolia theme
```

Comment:
This exact readme isn't necessary, but folders show up a lot nicer on GitHub if they have a readme.

Reply:
The thing is that you cannot run the examples separately. What do you want to put in the README then?

Reply:
A description of what this project is; similar to the little blurb on the website maybe? Since you can't run them individually, that's of course a bit of a problem. I wonder if we can get to a position where they can be run individually; but are bundled for the website at once

Reply:
Yes it's possible, it's Webpack so we can do whatever we want. We can add it when we have the needs.